### PR TITLE
Update CI build and add badges to readme

### PR DESCRIPTION
**Pull request by Levi-Armstrong**
Currently GitHub Actions does not support creating badges for individual jobs in a workflow, so this breaks up the CI into multiple workflows so individual badges can be inserted in the readme.

Also it switches the Windows CI Build from travis to GitHub Actions.

### CODE REVIEW

**Comment by Levi-Armstrong** [comment]
@marip8 Also include if you have time to review.

File: .github/workflows/xenial_build.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      OS_CODE_NAME: xenial
+      ROS_DISTRO: kinetic
+      ROS_REPO: main
+      TARGET_WORKSPACE: '. github:ros-industrial/cmake_common_scripts#master'
+      ROSDEP_SKIP_KEYS: "iwyu cmake_common_scripts"
+      CCACHE_DIR: "/home/runner/work/opw_kinematics/opw_kinematics/Xenial-Build/.ccache"
+      TARGET_CMAKE_ARGS: "-DCMAKE_BUILD_TYPE=Debug -DOPW_ENABLE_TESTING=ON"
+      AFTER_SCRIPT: 'catkin build -w $target_ws --no-deps --verbose opw_kinematics --make-args test'
+    steps:
+      - uses: actions/checkout@v1
```
Comment (marip8) [inline .github/workflows/xenial_build.yml:27]: Use checkout@v2

File: .github/workflows/focal_build.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      OS_CODE_NAME: focal
+      ROS_DISTRO: noetic
+      ROS_REPO: main
+      TARGET_WORKSPACE: '. github:ros-industrial/cmake_common_scripts#master'
+      ROSDEP_SKIP_KEYS: "iwyu cmake_common_scripts"
+      CCACHE_DIR: "/home/runner/work/opw_kinematics/opw_kinematics/Focal-Build/.ccache"
+      BEFORE_RUN_TARGET_TEST_EMBED: "source /root/target_ws/install/setup.bash"
+      TARGET_CMAKE_ARGS: "-DCMAKE_BUILD_TYPE=Debug -DOPW_ENABLE_TESTING=ON"
+    steps:
+      - uses: actions/checkout@v1
```
Comment (marip8) [inline .github/workflows/focal_build.yml:27]: Use checkout@v2

File: .github/workflows/clang_format.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    env:
+      CI_NAME: Clang-Format
+      OS_NAME: ubuntu
+      OS_CODE_NAME: bionic
+      ROS_DISTRO: melodic
+      ROS_REPO: main
+      CLANG_FORMAT_CHECK: file
+      CLANG_FORMAT_VERSION: 8
+    steps:
+      - uses: actions/checkout@v1
```
Comment (marip8) [inline .github/workflows/clang_format.yml:24]: Use checkout@v2

File: .github/workflows/bionic_build.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      OS_CODE_NAME: bionic
+      ROS_DISTRO: melodic
+      ROS_REPO: main
+      TARGET_WORKSPACE: '. github:ros-industrial/cmake_common_scripts#master'
+      ROSDEP_SKIP_KEYS: "iwyu cmake_common_scripts"
+      CCACHE_DIR: "/home/runner/work/opw_kinematics/opw_kinematics/Bionic-Build/.ccache"
+      TARGET_CMAKE_ARGS: "-DCMAKE_BUILD_TYPE=Debug -DOPW_ENABLE_TESTING=ON"
+      AFTER_SCRIPT: 'catkin build -w $target_ws --no-deps --verbose opw_kinematics --make-args test'
+    steps:
+      - uses: actions/checkout@v1
```
Comment (marip8) [inline .github/workflows/bionic_build.yml:27]: Use checkout@v2

**Review by marip8 — CHANGES_REQUESTED** [review]

**Comment by Levi-Armstrong** [comment]
@marip8 Update the checkout to v2.

**Review by marip8 — APPROVED** [review]

**Comment by Levi-Armstrong** [comment]
Thank you for the review @marip8

**Pull request merged**